src/cli/export.rs

```diff
@@ -40,18 +40,29 @@ pub async fn init(
 	// Initialize opentelemetry and logging
 	crate::telemetry::builder().with_log_level("error").init();
 
-	let root = Root {
-		username: &username,
-		password: &password,
+	let client = if let Some(username) = username {
+		let root = Root {
+			username: &username,
+			password: &password.expect("Password is required when username is provided"),
```

Comment:
`clap` argument parser can be configured to enforce this constraint (see [here](https://github.com/surrealdb/surrealdb/blob/860796ba867bbdc7061ea53743132399ddd91c86/src/cli/abstraction/mod.rs#L35) for an example). We still need `unwrap` but not a user-facing panic message.

Reply:
cool! didn't know it was possible. Much better know 👌